packages/flutter/test/cupertino/route_test.dart

```diff
@@ -309,23 +309,207 @@ void main() {
     expect(find.text('route'), findsNothing);
 
 
-    // Run the dismiss animation 75%, which exposes the route "push" button,
-    // and then press the button. MaterialPageTransition duration is 300ms,
-    // 275 = 300 * 0.75.
+    // Run the dismiss animation 60%, which exposes the route "push" button,
+    // and then press the button. A drag dropped animation is 400ms when dropped
```

Comment:
What is a "drag dropped animation" and where does 400ms come from?

Reply:
I moved this comment further down